common/common.go

```diff
@@ -198,8 +202,11 @@ func (l *PrivateNetList) Set(value string) error {
 		case len(netArgsPair) == 1:
 			l.mapping[netName] = ""
 		case len(netArgsPair) == 2:
-			if netName == "all" {
+			if netName == "all" ||
+				netName == "host" {
 				return fmt.Errorf("arguments are not supported by special netname %q", netName)
+			} else if netName == "none" {
```

Comment:
Isn't this impossible? You're setting netName to "default-restricted" if it's "none" above.